publication/ver11/accessibilty.md

```diff
@@ -0,0 +1,274 @@
+WoT does not constrain device behaviour and in particular does not prevent devices from flashing. 
+In the future, the way to handle this (which was not in the scope of our current charter) would be
+to define a standard model for lights (and other devices with visual output) that included this feature.
+### [n/a] It is possible to make navigation order correspond to the visual presentation.
+Not applicable.
+
+## If technology provides author control over color
+Generally, WoT does not mandate mechanisms for control of color, although some
+devices adhering to WoT may be able to display color.
+As stated above, WoT in general has no provision to constrain visual displays, 
+as this (and other controls limiting the behavior of devices)
+was not in the scope of our current charter.
+
+### [no] There is a mechanism for users to override colors of text and user interface components.	
+Not supported.
+### [no] There is a feature for authors to define semantically available "color classes" that users can easily map to custom colors, and give preference to this vs. coloring objects individually.
+Not supported.
+### [no] There is a feature for users to choose color schemata that work for them.	
+Not supported.
+### [no] The foreground and background color of an object can be reported to the user via AT.	
+Not supported.
+### [no] There are ways to set foreground and background colors separately for all objects.	
+Not supported.
+### [no] Compositing rules for foreground and background colors are well defined.	
+Not supported.
+
+## If technology provides features to accept user input 
+Devices adhering to WoT may in fact provide various forms of user input, such as buttons, dials,
+touch panels, etc.
+In general, WoT does not constrain the behavior of such devices but does make information
+available which can be used to enhance their accessibility.
+
+### [yes] There is a mechanism to label user input controls in an unambiguous and clear manner.
+WoT TDs have a "title" attribute for interactions
+which can be used to label an associated input control (but only if the device
+with the input control is capable of displaying text in association with the control, which is often
+no the case).
+### [yes] Authors can associate extended help information with a control.
+WoT TDs, in addition to "title", also have a "description" field which can contain extended text.
+In addition, a "link" can be used to hyperlink external documentation.
+### [partial] If there is an input error, it is possible to associate the error message clearly with the specific control that is in error.
+To the extent that interactions in TDs are associated with input controls, then an error on
+an interaction can be associated with that control.
+### [yes] There is a mechanism to report and set the state or value of controls programmatically.
+This is possible but not explictly required by WoT. A Thing representing an input device usually is 
```

Comment:
I'm not sure what is meant by this question.

Reply:
explained in the call.  In the browser context it means that JS can be used to, for instance, flip a button state, so that an alternative input device can be used.  Many IoT devices have something similar, and often this is their whole purpose.  For example, I have light switches in my house that I can flip manually, or under remote network control.  The latter is obviously useful if I could not use the manual interface for some accessibility reason and needed to set up an alternative interface.  Likewise getting the state is handy for mapping to other sensory modalities, for example reading the state of a light and stating it audibly (totally doable, AS LONG AS the light allows the current color to be read out; I have seen some that don't support this obvious feature!)